spec/components/previews/login_button_component_preview.rb

```diff
@@ -0,0 +1,27 @@
+class LoginButtonComponentPreview < ButtonComponentPreview
+  # @!group Preview
+  def default
+    render(LoginButtonComponent.new.with_content('Sign in with'))
+  end
+
+  # @!endgroup
+
+  # rubocop:disable Layout/LineLength
+  # @param content text
+  # @param big toggle
+  # @param color select [~,primary,primary darker,primary lighter]
+  def workbench(
+    content: 'Sign in with',
```

Comment:
I'd suggest parameterizing this string with `%{app_name}`. I'm assuming it's configurable to support localization, and we shouldn't assume that a translated text will always append "Login.gov" at the end of the string.

```suggestion
    content: 'Sign in with %{app_name}',
```

This might make the interpolation a little more tricky with handling the screen reader text HTML, but we have some other places where we interpolate HTML strings into content, e.g.

https://github.com/18F/identity-idp/blob/b733821dea53bbec5aeb006c393d29a153ce3dbd/app/presenters/idv/otp_verification_presenter.rb#L13-L19

If we need to interpolate after the string is already translated, maybe we can use `I18n.interpolate` directly.

Something like...

```rb
I18n.interpolate(content, app_name: content_tag(:span, APP_NAME, class: 'usa-sr-only')
```

(There might be some complications with default string HTML sanitization)